rest_framework/exceptions.py

```diff
@@ -95,6 +95,12 @@ def __hash__(self):
         return hash(str(self))
 
 
+class ObjectValueError(Exception):
+    """
+    A value was not suitable for requesting an object.
```

Comment:
I think this could have a clearer docstring. Mention that we use it to indicate when invalid values are passed to ‘queryset.get()’.

Reply:
Are the new docstrings more clear? Generally, these exceptions will only be caught internally, so it should be fine?